lib/gds_api/test_helpers/calendars.rb

```diff
@@ -1,8 +1,7 @@
 module GdsApi
   module TestHelpers
     module Calendars
-      CALENDARS_ENDPOINT = Plek.current.find('calendars')
-
+      CALENDARS_ENDPOINT = Plek.new.website_root
```

Comment:
Is it worth renaming this constant since it no longer represents the actual endpoint? or maybe just move `Plek.new.website_root` into `stub_calendars_endpoint` instead?

Reply:
Sounds good, I've pushed up an amended commit to put it directly into `stub_calendars_endpoint`.